### DISCUSSION TITLE
Retrying sync HTTP tasks with exponential backoff

### DISCUSSION THREAD

**Discussion by coderrr22** [Netflix/conductor#3204] [Q&A]
We are using conductor version v3.10.6 with REDIS as queue.
We have 25 replicas of 1 cpu 1gb memory conductor servers running on the cloud.
When I'm trying to run 6000 workflows that contain simple HTTP tasks only ,in parallel 
The scheduling time for each http task can reach up to 2 minutes.
Which is way too slow for our needs.

Hence I've read [here](https://github.com/Netflix/conductor/issues/1183#issuecomment-503246838)  That I can change the HTTP task def to is async false.
Ive tried it and indeed it made the entire operation run 2 times faster without any problems.

Until I noticed It's not possible to use exponential backoff retry for those http tasks now that they have async false.
I googled around and found this thread talking about this issue [here](https://github.com/Netflix/conductor/issues/1059)
Which is required for us.

So how/where could I change Netflix conductor code to support exponential backoff for sync http tasks.

Or any other recommendation to make the system faster with support for exponential backoff will be welcome.
We are using elastic version 6.8 with 3 strong replicas.

Thanks!

**Comment by apanicker-nflx** [comment]
Consider defining a task definition for your HTTP tasks with the retry count and retry strategy as desired.

**Reply by rajeshwar-nu** [reply]
Can you create task definition for System Tasks? As per these docs, Task Definitions are only used to register SIMPLE tasks https://conductor.netflix.com/documentation/configuration/taskdef.html
